src/script/ptl-tool.py

```diff
@@ -120,49 +193,81 @@ def build_branch(branch_name, pull_requests):
 
         if new_new_contributors:
```

Comment:
> We are going to run this script in a test machine, mostly in smithi, without the intention to update the contributors list.

Idea is if we have a reviewer we haven't seen before, then the merger should figure out who they are (if possible, skip using EOF/CTRL+D if not) and add them.

>I think it will make .githubmap a mess, as any text is acceptable here.

We can add warnings but I don't really think this is a big deal.

Reply:
@batrick 
Agreed.

> Need name for contributor "pagupta"; Reviewed-by:
I was wondering, if it is possible to `.get` the Name and Email ID from github. Let me check if that is possible.

Reply:
I don't think it is possible to implement it correctly, as a user can choose not to specify his email address as a Public Email. 

The user can also make his email address private:
"
Keep my email address private
We'll remove your public profile email and use 24334514+joscollin@users.noreply.github.com when performing web-based Git operations and sending email on your behalf. 
"

However, I have sent a query to github support to confirm this.